algebird-test/src/test/scala/com/twitter/algebird/HyperLogLogTest.scala

```diff
@@ -152,7 +152,7 @@ class HyperLogLogTest extends WordSpec with Matchers {
       }
     }
     "Correctly serialize" in {
-      (4 to 20).foreach { bits =>
+      (4 to 31).foreach { bits =>
```

Comment:
going to 31 is probably going to OOM no matter what for dense HLLs. You might want to only go to 29 or 28 or something, something just large enough to exercise this new code.